CONTRIBUTING.md

```diff
@@ -65,7 +65,7 @@ rules on your computer. We will now walk you through the steps to get going:
 
 1.  Install VirtualBox by [downloading and running an
     installer](https://www.virtualbox.org/wiki/Downloads), or
-    [Docker|https://www.docker.com/].
+    [Docker](https://www.docker.com/).
```

Comment:
Ah from https://minikube.sigs.k8s.io/docs/drivers/ I understand that docker is available as a driver, and even preferred while virtualbox isn't preferred.

Have you tried getting kubespawner setup with minikube and docker as a driver to create the k8s cluster and verified that you could run the kubespawner test suite against it successfully?